packages/gatsby/src/utils/api-runner-node.js

```diff
@@ -90,6 +90,9 @@ const runAPI = (plugin, api, args) => {
     const namespacedCreateNodeId = id => createNodeId(id, plugin.name)
 
     const tracing = initAPICallTracing(pluginSpan)
+    const cache = new Cache({
+      name: plugin.name,
+    }).init()
```

Comment:
not sure if that matters -this will init cache for every API we run - so for markdown benchmark with 10000 pages - it will run init remark cache over 10000 times (for each `onCreateNode` run plus some single things like `createPages` etc)

Reply:
All I need is the plugin name. So the _earliest_ point we know that would be perfect for this PR, because then I can re-use the cache instance.